### PR TITLE
docs: add publish script

### PR DESCRIPTION
**Pull request by mattklein123**

### CODE REVIEW

**Comment by mattklein123** [comment]
@lyft/network-team

File: docs/publish.sh [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+
+git -C $PUBLISH_DIR fetch
+git -C $PUBLISH_DIR checkout -B gh-pages origin/gh-pages
+rm -fr $PUBLISH_DIR/*
+cp -r $DOCS_DIR/* $PUBLISH_DIR
+git -C $PUBLISH_DIR add .
+git -C $PUBLISH_DIR commit -m "docs @$BUILD_SHA"
+
+echo
+echo "*** YOU MUST MANUALLY PUSH the gh-pages branch after verifying the commit ***"
```
Comment (RomanDzhabarov) [inline docs/publish.sh]: do i need to do anything else after push to origin?

can you put in some docs that, `make publish_docs` should be enough or we need to do something else

Comment (mattklein123) [inline docs/publish.sh]: Added README, let me know if that looks OK

File: docs/conf.py [hunk 2]
```diff
@@ -55,11 +56,14 @@
...
+
+repo = git.Repo('../')
+last_commit = str(repo.head.commit)[:6]
+
 # The short X.Y version.
-version = u'1.0.0'
+version = u'1.1.0-' + last_commit
 # The full version, including alpha/beta/rc tags.
-release = u'1.0.0'
+release = u'1.1.0' + last_commit
```
Comment (RomanDzhabarov) [inline docs/conf.py]: do not much care about these, curious about the  reason of making version as 0-sha and release  as 0sha

Comment (mattklein123) [inline docs/conf.py]: typo fixing

**Pull request merged**